sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -997,6 +997,32 @@ class CollationSQLExpressionsSuite
     })
   }
 
+  test("Support CurrentDatabase/Catalog/User expressions with collation") {
+    // Supported collations
+    Seq("UTF8_BINARY_LCASE", "UNICODE", "UNICODE_CI").foreach(collationName =>
+      withSQLConf(SqlApiConf.DEFAULT_COLLATION -> collationName) {
+        val queryDatabase = sql("SELECT current_schema()")
+        val queryCatalog = sql("SELECT current_catalog()")
+        val queryUser = sql("SELECT current_user()")
+        // Result
```

Comment:
This only changes return types and we should only test data types, not results. We should avoid duplicated tests as it introduces behavior changes when we make behavior changes in the future and need to update tests.

Reply:
makes sense, fixing now - thanks